cli/bin/popper

```diff
@@ -16,6 +16,7 @@ else:
 sys.path.insert(0, yaml_lib)
 if sys.version_info[0] < 3:
     sys.path.insert(0, os.path.join(vendordir, 'python-future', 'src'))
+    sys.path.insert(0, os.path.join(vendordir, 'pythonfutures', 'src'))
```

Comment:
same as above

Reply:
@JayjeetAtGithub this does not work because the `pythonfeatures` repo does not have a `src` folder. The library is at the root of the project